labs/lab-2-javascript-game/javascript.js

```diff
@@ -0,0 +1,81 @@
+var userHealth = 40;
+var grantHealth = 10;
+var wins = 0;
+var playGame = true;
+var battleName = "";
+
+function startGame() {
+  var playGame = confirm("Shall we battle?");
+  if (playGame === true) {
+    battleTitle();
+    startCombat();
+  } else alert("Return when you have obtained the courage to battle!!");
+}
+startGame();
+
+function battleTitle() {
+  var name = prompt("Enter your battle name!");
+  return battleName = name;
```

Comment:
As we discussed last night, just `return name;` or even `return prompt("Enter your battle name!");`

It's good you're breaking things into small functions, but your conceptual understanding of functions is still soggy. Crisp it up -- maybe read (or re-read) the functions section of your textbook!